tests/test_models/test_simple_models.py

```diff
@@ -558,6 +556,21 @@ def test_get_model_after_training(example_tsds, etna_model_class, expected_class
         assert isinstance(models_dict[segment], expected_class)
 
 
+@pytest.mark.parametrize(
+    "etna_model_class",
+    (
+        NaiveModel,
+        SeasonalMovingAverageModel,
+        MovingAverageModel,
+    ),
+)
+def test_get_model(example_tsds, etna_model_class):
```

Comment:
Why can't we just improv the previous tests?

Reply:
I think that in #1117 we will just merge this two.